website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,6 +9,10 @@ description: >-
 
 Keep track of changes to the API for HCP Terraform and Terraform Enterprise.
 
+## 2024-07-18
+* Add `default-project` relationship to the [`organization` API](/terraform/cloud-docs/api-docs/organizations).
+* Fix `default-agent-pool` data type from attribute to relationship in the [`organization` API](/terraform/cloud-docs/api-docs/organizations) documentation.
```

Comment:
I think we can remove this from the changelog because it does not represent a change to the API itself.

Reply:
I will remove!